### ISSUE TITLE
Message preview in message list sometimes gets misrepresented

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#8222]
### Steps to reproduce

1. View the message list in any mailbox
2. Sometimes it looks off (listed below)

### Expected behavior

The preview text should look like the text when you open the mail.

### Actual behavior

Sometimes the text includes:
- ALL-CAPS of text which is normally written in the original mail (like for our own notifications, see screenshot below)
- Alt-text of images in `[Brackets]`, unnecessary inclusion (see below)
- Links rendered in `[Markdown style](https://example.com)`, also unnecessary to include the link part, only the text is enough.

![image](https://user-images.githubusercontent.com/925062/224954608-bcdbf46d-c236-45b2-ba35-560217abc556.png)

### Mail app version

Current on cloud.nc.com

### PHP engine version

None

### Web server

None

### Database

None

**Comment by hamza221** [comment]
Can you reproduce locally with the latest main?

**Comment by GretaD** [comment]
> Can you reproduce locally with the latest main?

Yes, that is there, but i am having a look if the text insde [] is added by a sysadmin setup like [external] or mail app adds it

**Comment by hamza221** [comment]
After a bit of investigation, the issue comes from the html2text package. The link problem is easily fixable since html2text provides an option to hide markdown-style links. when it comes to image alt text, If https://github.com/mtibben/html2text/pull/116 gets merged the problem will be fixed, if not a workaround would also be possible. I'm not sure I understood 
> * ALL-CAPS of text which is normally written in the original mail (like for our own notifications, see screenshot below)
correctly, So we should avoid having all-caps text even if it's in the original mail?

**Comment by hamza221** [comment]
Also, In the case of a list the preview looks like this 
<img width="354" alt="image" src="https://user-images.githubusercontent.com/40746210/225627605-fe2f52c7-6840-420d-b3a0-7157ef207dab.png">
Should we keep it the way it is, or remove the asterisks

**Comment by ChristophWurst** [comment]
In the specific case of a suspicious_login email (and likely other nextcloud emails) there are two alternative parts in the message. A text one and a HTML one. I probably makes most sense to take the text part for the preview, not the HTML.

![Bildschirmfoto vom 2023-03-16 14-15-03](https://user-images.githubusercontent.com/1374172/225628224-239b3a40-2b80-4891-9b82-2e904455ed12.png)

**Comment by hamza221** [comment]
> After a bit of investigation, the issue comes from the html2text package. The link problem is easily fixable since html2text provides an option to hide markdown-style links. when it comes to image alt text, If [mtibben/html2text#116](https://github.com/mtibben/html2text/pull/116) gets merged the problem will be fixed, if not a workaround would also be possible. I'm not sure I understood
> 
> > * ALL-CAPS of text which is normally written in the original mail (like for our own notifications, see screenshot below)
> 
> correctly, So we should avoid having all-caps text even if it's in the original mail?

To circle back on this, preview text should always be lowercase even when the html contains a heading`<h*> ?
